include/bluetooth/conn.h

```diff
@@ -328,6 +328,14 @@ struct bt_conn_le_create_param {
 	 *  Set zero to use same as LE 1M PHY scan window.
 	 */
 	u16_t window_coded;
+
+	/** Connection initiation timeout
+	 *
+	 *  Set zero to use the default timeout:
+	 *      K_SECONDS(:option:`CONFIG_BT_CREATE_CONN_TIMEOUT`)
+	 *  @note Unused in @ref bt_conn_create_auto_le
```

Comment:
Unsure if this affects the doxygen or not. But we usually have a blank comment line before the note.